x-pack/plugins/siem/server/graphql/timeline/schema.gql.ts

```diff
@@ -0,0 +1,228 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import gql from 'graphql-tag';
+
+const columnHeader = `
+  aggregatable: Boolean
```

Comment:
Is it necessary to allow `null` for most of these attributes? I noticed most of the arrays below and `PageInfoTimeline` have the explicit `!` to prevent that. Just curious if all these null-values are needed/intended.

Reply:
As right now, you are not always saving the column header. it can just be the title.

after going this route, I really want to refactor it to have one type for mutation and query and apply the same thing in the io-ts. It will remove a lot of weird type in the code.